src/outputs.c

```diff
@@ -225,6 +225,19 @@ void deinit_outputs() {
         evdev = NULL;
     }
 }
+void singletap(uint32_t timestamp_ms, imu_quat_type quat, imu_euler_type velocities, bool ipc_enabled,
+                       bool imu_calibrated, ipc_values_type *ipc_values) {
+    libevdev_uinput_write_event(uinput, EV_KEY, BTN_LEFT,1);
+    libevdev_uinput_write_event(uinput, EV_SYN, SYN_REPORT, 0);
+    libevdev_uinput_write_event(uinput, EV_KEY, BTN_LEFT,0);
+    libevdev_uinput_write_event(uinput, EV_SYN, SYN_REPORT, 0);
+}
+
+void doubletap(uint32_t timestamp_ms, imu_quat_type quat, imu_euler_type velocities, bool ipc_enabled,
+                       bool imu_calibrated, ipc_values_type *ipc_values) {
+    singletap(timestamp_ms, quat, velocities, ipc_enabled, imu_calibrated, ipc_values);
+    singletap(timestamp_ms, quat, velocities, ipc_enabled, imu_calibrated, ipc_values);
+}
 
 void handle_imu_update(uint32_t timestamp_ms, imu_quat_type quat, imu_euler_type velocities, bool ipc_enabled,
```

Comment:
If we move forward with this, I'm thinking we could actually just modify this signature with a param like `int tapsDetected` and then you'd be able to move the single/double tap stuff into the `is_mouse_mode` section that's already here.